app/src/androidTest/java/org/mozilla/tv/firefox/ui/ClearSessionTest.kt

```diff
@@ -42,46 +44,33 @@ class ClearSessionTest {
 
     @Test
     fun WHEN_data_is_cleared_THEN_back_and_forward_should_be_unavailable() {
-
         val endpoints = MockServerHelper
                 .initMockServerAndReturnEndpoints("This is Google", "This is YouTube")
 
         home {
             assertCanGoBackForward(false, false)
 
-            navigateToPage(endpoints[0])
-
-            openMenu()
-
-            assertCanGoBack(true)
-
-            navigateToPage(endpoints[1])
-
-            openMenu()
+        }.enterUrlAndEnterToBrowser(endpoints[0]) {
+        }.openOverlay {
```

Comment:
I find this test a little difficult to read as it currently stands, but after playing around with it I haven't found any formatting that I like better.  I think we should get a third opinion on this file before we merge this in.

Reply:
@liuche Any thoughts on making this more readable? If so, please file a new bug to follow-up.

Reply:
Looking at the test in-tree, I think it looks fine - each block is a separate step, and the indentation makes it easy to understand the higher-level flow, so I'm fine with it.